### ISSUE TITLE
Grid disappeared after version update

### ISSUE THREAD

**Issue by dawidbak** [glideapps/glide-data-grid#461]
Hi, after upgrading to 5.0 there have been big changes to my grid. Namely, my whole grid disappeared and a text box with buttons appeared. How can I set up to go back to the "old" grid?

Before:
![image](https://user-images.githubusercontent.com/75274385/183415279-f1294949-a85c-41d3-ab77-601728e56211.png)

After:
![image](https://user-images.githubusercontent.com/75274385/183415338-77cd5ee8-1f60-4786-a464-8126a8552398.png)

**Comment by jassmith** [comment]
Did you check the migration guide? https://github.com/glideapps/glide-data-grid/releases/tag/v5.0.0 You didn't include the CSS

**Comment by dawidbak** [comment]
Yes, you are right i missed somehow that i need to import new css. Thank you so much
